doc/spinel-protocol-src/spinel-prop-core.md

```diff
@@ -239,6 +245,59 @@ Unlike most other properties, setting this property to true when the
 value of the property is already true **MUST** fail with a last status
 of `STATUS_ALREADY`.
 
+### PROP 10: PROP_HOST_POWER_STATE {#prop-host-power-state}
+
+* Type: Read-Write
+* Packed-Encoding: `C`
+* Default value: 4
+
+Octets: |        1
+--------|------------------
+Fields: | `HOST_POWER_STATE`
+
+Describes the current power state of the *host*. This property is used
+by the host to inform the NCP when it has changed power states. The
+NCP can then use this state to determine which properties need
+asynchronous updates. Enumeration is encoded as a single unsigned
+byte. These states are defined in similar terms to `PROP_POWER_STATE`
+((#prop-power-state)).
+
+Defined values are:
+
+*   0: `HOST_POWER_STATE_OFFLINE`: Host is physically powered off and
+    cannot be woken by the NCP. All asynchronous commands are
+    squelched.
+*   1: `HOST_POWER_STATE_DEEP_SLEEP`: The host is in a low power state
+    where it can be woken by the NCP but will potentially require more
+    than two seconds to become fully responsive.
+*   2: *RESERVED*. This MUST NOT be sent by the host. If received by
+    the NCP, the NCP SHOULD consider this as a synonym of
+    `HOST_POWER_STATE_DEEP_SLEEP`.
+*   3: `HOST_POWER_STATE_LOW_POWER`: The host is in a low power state
+    where it can be immediately woken by the NCP. The NCP SHOULD avoid
+    sending unnecessary property updates, such as child table updates
+    or non-critical messages on the debug stream.
+*   4: `HOST_POWER_STATE_ONLINE`: The host is awake and responsive. No
+    special filtering is performed by the NCP on asynchronous updates.
+*   All other values are *RESERVED*. They MUST NOT be set by the host.
+    If received by the NCP, the NCP SHOULD consider the value as a synonym
+    of `HOST_POWER_STATE_LOW_POWER`.
+
+<!-- RQ
+  -- We should consider reversing the numbering here so that 0 is
```

Comment:
Will do that when we renumber, since changing it will require a protocol version change.